lib/qtermwidget_interface.h

```diff
@@ -99,9 +99,11 @@ class QTermWidgetInterface {
    virtual void setBoldIntense(bool boldIntense) = 0;
    virtual void setConfirmMultilinePaste(bool confirmMultilinePaste) = 0;
    virtual void setTrimPastedTrailingNewlines(bool trimPastedTrailingNewlines) = 0;
+   virtual QString wordCharacters() const = 0;
+   virtual void setWordCharacters(const QString& chars) = 0;
    virtual QTermWidgetInterface* createWidget(int startnow) const = 0;
 };
 
-#define QTermWidgetInterface_iid "lxqt.qtermwidget.QTermWidgetInterface/1.0"
+#define QTermWidgetInterface_iid "lxqt.qtermwidget.QTermWidgetInterface/1.5"
```

Comment:
Is this necessary? I guess adding new functions does not break backward compatibility and the IID can remain the same.

Reply:
I'm more worried about _forward_ compatibility: A program that wants to use the new functions that imports an older version of the plugin will crash.

Reply:
Yes, that makes sense.